### PR TITLE
Fix hex input overriding alpha value

### PR DESCRIPTION
**Pull request by VLuisa**
Fix for https://github.com/mapbox/studio/issues/13487

### QA steps:
- [x] Set alpha value
- [x] Change color via hex input
- [x] Alpha should be the same as it was

@tristen for review

### CODE REVIEW

**Review by tristen — CHANGES_REQUESTED** [review]
Yay! This works great! It's just missing a test that looks a little like this one: https://github.com/mapbox/react-colorpickr/blob/361a8e3660fde73e213ce77f8acfd18ae415ccf1/src/colorpickr.test.js#L22-L45

I think if you pass an `initialValue` for props like https://github.com/mapbox/react-colorpickr/blob/361a8e3660fde73e213ce77f8acfd18ae415ccf1/src/colorpickr.test.js#L290, you can assert the alpha value is retained.

File: src/colorpickr.test.js [hunk 1]
```diff
@@ -116,6 +116,33 @@ describe('Colorpickr', () => {
...
+      fireEvent.change(input, mockEvent);
+      expect(props.onChange).toHaveBeenCalledTimes(1);
+      expect(props.onChange).toHaveBeenCalledWith({
+        h: 0,
+        s: 0,
+        l: 20,
+        r: 51,
+        g: 51,
+        b: 51,
+        a: 1,
```
Comment (tristen) [inline src/colorpickr.test.js]: 🤔 hrm shouldn't this be `0.5`?

Comment (VLuisa) [reply src/colorpickr.test.js]: 🙃 oops. yes it should be! will revisit, don't think I'm setting this up correctly

**Review by tristen — APPROVED** [review]
🌈

**Pull request merged**